### DISCUSSION TITLE
cve-2023-43804 vulenrability in strimzi 0.40.0 operator

### DISCUSSION THREAD

**Discussion by git-azeez** [strimzi/strimzi-kafka-operator#10069] [Q&A]
Can this vulnerability be ignored.

**Comment by scholzj** [comment]
You should probably start by explaining what the CVE is, which parts of Strimzi are affected by this, which subprojects/modules, which container images, which paths inside them etc.

**Reply by git-azeez** [reply]
I found this vulnerability on that docker image , with a description as python3-pip-wheel. Apart from this there is no any lead information
